test/e2e/test-scripts/place-bid-by-discount.sh

```diff
@@ -3,5 +3,5 @@
 
 source ./test/e2e/test-scripts/common.sh
 
-agops inter bid by-discount --discount 5 --give 2IST --from $accountAddress --keyring-backend=test 2>&1
+agops inter bid by-discount --discount 5 --give 2IST --from $user2Address --keyring-backend=test 2>&1
```

Comment:
`2>&1` doesn't seem appropriate here - `inter bid` sends data (in JSON) to stdout and misc. progress info to stderr.

Oh... but I don't see anything consuming any of the output. So maybe it doesn't matter.

Reply:
ah. I see the .js code consumes the output.